### DISCUSSION TITLE
Android 12 / 10以上，偶现Scudo ERROR: corrupted chunk header崩溃

### DISCUSSION THREAD

**Discussion by PJYGit** [Tencent/libpag#1643] [Q&A]
https://github.com/Tencent/libpag/issues/1639
https://github.com/Tencent/libpag/issues/1640

两个偶现崩溃，看起来都跟memory分配有关，大佬们有想法吗？是多线程问题还是什么

**Comment by kevingpqi123** [comment]
关闭重复问题
